### PR TITLE
Use OpenRISC pipeline builder from pipeline library

### PR DESCRIPTION
**Pull request by Nancy-Chauhan**
- [x] Use pipeline builder from https://github.com/librecores/librecores-pipeline-lib/pull/5
- [x] Add Yosys reporting to pipeline

### CODE REVIEW

**Comment by stffrdhrn** [comment]
so far it looks good

**Comment by Nancy-Chauhan** [comment]
Hello, @stffrdhrn @oleg-nenashev changes done!  
With this, Yosys should run successfully, tested on my local instance

File: Jenkinsfile [hunk 1]
```diff
@@ -1,4 +1,11 @@
 openriscPipeline {
+
+    yosysReport {
+		core 'mor1kx'
+		target 'synth'
+		logPath 'build/mor1kx_5.0-r3/synth-icestorm/yosys.log'
```
Comment (stffrdhrn) [inline Jenkinsfile]: again here we can't keep the version in the path name.

Comment (Nancy-Chauhan) [reply Jenkinsfile]: @stffrdhrn This should work now. @oleg-nenashev will be looking into https://ci.librecores.org/ stability maybe in a couple of weeks

**Pull request merged**